src/WorkflowCore/Services/BackgroundTasks/WorkflowConsumer.cs

```diff
@@ -55,7 +55,7 @@ protected override async Task ProcessItem(string itemId, CancellationToken cance
                     finally
                     {
                         WorkflowActivity.Enrich(result);
-                        await _persistenceStore.PersistWorkflow(workflow, result.Subscriptions, cancellationToken);
+                        await _persistenceStore.PersistWorkflow(workflow, result.Subscriptions);
```

Comment:
It seems like this line is the only thing that needs to change to achieve your need?